bleach/html5lib_shim.py

```diff
@@ -45,19 +45,119 @@
 CHARACTERS_TYPE = constants.tokenTypes['Characters']
 
 
-#: List of HTML tags
+#: List of valid HTML tags
```

Comment:
If we're going to have a comprehensive list of tags, we should specify how it's comprehensive. Is this the complete list of tags defined in the HTML 5.3 spec or something else?

Reply:
Sure - I had taken it from MDN originally, but I've updated it slightly now to use the list from the WHATWG spec, and added a link to their list in their index. (I'm not sure what the `#:` comments indicate, so I don't know if my new line should have a colon prefix too).

Let me know how that seems or if you'd like something else.